chain-config.ts

```diff
@@ -1,26 +1,29 @@
-export default {
-  "plasma": {
-    "dappchainEndpoint" : "https://plasma.dappchains.com",
-    "mainnetEndpoint" : "https://mainnet.infura.io",
-    "chainId" : "default",
-    "gatewayAddress" : "0x8f8e8b3c4de76a31971fe6a87297d8f703be8570"
+import { DahboardEnvs } from "@/types"
+const configs = {
+  plasma: {
+    dappchainEndpoint : "wss://plasma.dappchains.com",
+    mainnetEndpoint : "https://mainnet.infura.io",
+    chainId : "default",
+    gatewayAddress : "0x8f8e8b3c4de76a31971fe6a87297d8f703be8570",
   },
-  "asia1": {
-    "dappchainEndpoint" : "https://test-z-asia1.dappchains.com",
-    "mainnetEndpoint" : "https://rinkeby.infura.io",
-    "chainId" : "default",
-    "gatewayAddress" : "0x76c41eFFc2871e73F42b2EAe5eaf8Efe50bDBF73"
+  asia1: {
+    dappchainEndpoint : "https://test-z-asia1.dappchains.com",
+    mainnetEndpoint : "https://rinkeby.infura.io",
+    chainId : "default",
+    gatewayAddress : "0x76c41eFFc2871e73F42b2EAe5eaf8Efe50bDBF73",
   },
-  "us1": {
-    "dappchainEndpoint" : "https://test-z-us1.dappchains.com",
-    "mainnetEndpoint" : "https://rinkeby.infura.io",
-    "chainId" : "default",
-    "gatewayAddress" : "0xEA319a0Ea64f482060032b4BE8d9d3F7232c1214"
+  us1: {
+    dappchainEndpoint : "wss://test-z-us1.dappchains.com",
+    mainnetEndpoint : "https://rinkeby.infura.io",
+    chainId : "default",
+    gatewayAddress : "0x76c41eFFc2871e73F42b2EAe5eaf8Efe50bDBF73",
```

Comment:
The address was correct
```suggestion
    gatewayAddress : "0xEA319a0Ea64f482060032b4BE8d9d3F7232c1214",
```